src/css/components/_c-person.scss

```diff
@@ -123,6 +134,24 @@ $_person-distance: (
 	&:hover {
 		text-decoration: none;
 	}
+
+	// User Queries
+	@media (prefers-color-scheme: dark) {
+		@include link-states($color-blue-tint-darkmode);
+	}
+}
+
+
+.c-person__spotlight-link {
+	@include link-states($color-blue-shade-lightmode);
+
+	display: inline-block;
+	margin-top: 1rem;
+
+	// User Queries
+	@media (prefers-color-scheme: dark) {
+		@include link-states($color-blue-tint-darkmode);
+	}
```

Comment:
Similar to above, $color-blue-shade-lightmode seems to return a color hex and not color name. Hence this results in `--color-#232d71` which gives an error and doesn't render correctly. Can be seen in spotlight page. Same for the dark mode.